gnat2goto/driver/tree_walk.adb

```diff
@@ -3541,6 +3544,32 @@ package body Tree_Walk is
       return R;
    end Do_Op_Or;
 
+   -------------------------
+   --      Do_Op_And      --
+   -------------------------
+
+   function Do_Op_And (N : Node_Id) return Irep is
+      LHS_Bool_Value : constant Irep := Do_Expression (Left_Opnd (N));
+      RHS_Bool_Value : constant Irep := Do_Expression (Right_Opnd (N));
+      Cast_LHS_To_Integer : constant Irep :=
+        Make_Op_Typecast (Op0 => LHS_Bool_Value,
+                          Source_Location => Sloc (N),
+                          I_Type => Make_Signedbv_Type (Make_Nil_Type, 32));
+      Cast_RHS_To_Integer : constant Irep :=
+        Make_Op_Typecast (Op0 => RHS_Bool_Value,
+                          Source_Location => Sloc (N),
+                          I_Type => Make_Signedbv_Type (Make_Nil_Type, 32));
```

Comment:
Not this PRs fault, but it's _very_ weird that `Make_Signedbv_Type` takes subtype indications. I'll check if this can be removed.